cpp/ql/src/jsf/4.10 Classes/AV Rule 79.ql

```diff
@@ -191,9 +191,7 @@ predicate freedInSameMethod(Resource r, Expr acquire) {
   exists(Expr releaseExpr, string kind |
     r.acquisitionWithRequiredKind(acquire, kind) and
     releaseExpr = r.getAReleaseExpr(kind) and
-    releaseExpr.getEnclosingFunction().getEnclosingAccessHolder*() = acquire.getEnclosingFunction()
-    	// here, `getEnclosingAccessHolder*` allows us to go from a nested function or lambda
-    	// expression to the class method enclosing it.
+    releaseExpr.getEnclosingElement*() = acquire.getEnclosingFunction()
```

Comment:
One detail: this `*` can be a `+`. It's probably insignificant here, but I've been bitten by this before -- see 11e03b31611d3ef6a5ddece7c10e868951ac5e69.

Reply:
Done.